src/index.js

```diff
@@ -1,29 +1,95 @@
 import './style.css';
 
-const todoListArr = [
-  {
-    description: 'Take the car to the mechanic',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Attend choir practice at 6:00 PM',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Do some coding challenges on data structures',
-    completed: false,
-    index: 3,
-  },
-];
-
-todoListArr.forEach((todo) => {
-  const todoContent = `
-  <div>
-  <input type="checkbox" class=checkbox>
-  <span>${todo.description}</span>
+const todoListSection = document.getElementById('todo-list');
+const todoText = document.querySelector('input');
+
+class TodoObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const removeTodoList = (todo) => {
+  todoListSection.removeChild(todo);
+  const localdata = JSON.parse(localStorage.getItem('todolist'));
+  const data = Array.from(localdata).filter((i) => i.completed === false);
+  data.map((i) => i.index);
+  localStorage.setItem('todolist', JSON.stringify(data));
+};
+
+const editTodoList = (todoContainer, todoDescription) => {
+  const editInputText = document.createElement('input');
+  editInputText.type = 'text';
+  editInputText.classList = 'editInput';
+  editInputText.value = todoDescription.textContent;
+  todoContainer.replaceChild(editInputText, todoDescription);
+  editInputText.addEventListener('keypress', (e) => {
```

Comment:
- Kindly update your UI so that the ellipses icon or delete icon are always aligned to the right as per design specifications.
![image](https://user-images.githubusercontent.com/38088818/173178817-f17a50c7-a9cd-4025-b70c-edb71317a6dd.png)